sdk/python/feast/cli.py

```diff
@@ -99,7 +99,7 @@ def cli(
     ctx.obj["FS_YAML_FILE"] = (
         Path(feature_store_yaml).absolute()
         if feature_store_yaml
-        else ctx.obj["CHDIR"] / "feature_store.yaml"
+        else utils.get_default_yaml_file_path(ctx.obj["CHDIR"])
```

Comment:
nice! I think you're missing a few spots in the codebase where you could also introduce this change

if you search for '"feature_store.yaml"' you should fine all of them, e.g. there's one in `infra/aws.py`

Reply:
also do you mind adding a test? I would take a look at `test_cli.py`, specifically you should be able to modify `test_3rd_party_registry_store_with_fs_yaml_override` to test your change pretty easily!

Reply:
@felixwang9817, thank you for your suggestions,
I've also introduced changes in aws.py and lambda_engine.py and added one test into test.cli.py